docs/samples/openai/basic-semantic-kernel-integration/README.md

```diff
@@ -0,0 +1,42 @@
+# Integration with Semantic Kernel Sample
```

Comment:
Please add this sample to the docs/samples/python/index.md so that it is accessible.
https://github.com/microsoft/presidio/blob/main/docs/samples/python/index.md